kmath-histograms/src/commonMain/kotlin/space/kscience/kmath/histogram/Histogram.kt

```diff
@@ -32,29 +32,38 @@ public interface Histogram<in T : Any, out B : Bin<T>> {
     public val dimension: Int
 
     public val bins: Iterable<B>
+
+    public companion object {
```

Comment:
Is it used?

Reply:
Yes, it is a discoverability root. All Histogram builders are attached to it.